vrpn_Mutex.C

```diff
@@ -1448,9 +1448,10 @@ void vrpn_PeerMutex::init(const char *name)
 {
     d_mutexName = NULL;
     try {
-      d_mutexName = new char[1 + strlen(name)];
+      size_t n = 1 + strlen(name);
+      d_mutexName = new char[n];
       // This is guaranteed to fit because of the new allocation above.
-      strncpy(d_mutexName, name, strlen(name));
+      strncpy(d_mutexName, name, n);
```

Comment:
This will fail to copy the terminating null character to the end of the string; it needs to be set back to n to make it copy that character.

Reply:
I thought `n = 1 + strlen` would mean that `n` includes the null.

Reply:
You're right.  I was looking at the replaced line as if it were the new line.  Doh!